apps/aevm/lib/aevm.ex

```diff
@@ -0,0 +1,1430 @@
+defmodule Aevm do
```

Comment:
can you add one line names/descriptions to each function/patternmatch

Reply:
I don't know it this isn't maybe considered resolved, but I wouldn't split. This file contains only one function so you don't have to be able to easily read whole file, because you know exactly whats in it without reading it. You can also easily find every function. It's also probably more efficient to do this this way.